File: C/03 - oddOrEven.c
#include <stdio.h>

int main(){
    int number;
    printf("Enter any number: ");
    scanf("%d", &number);
    if (number % 2 == 0) { printf("%d is even number", number); }
    else { printf("%d is odd number", number); }
    return 0;
}

File: C#/03 - oddOrEven.cs
using System;

public class sumUptoNTerms
{
    public static void Main(string[] args)
    {
        int number;
        Console.Write("Enter any number: ");
        number = Convert.ToInt32(Console.ReadLine());
        if (number % 2 == 0) { Console.WriteLine($"{number} is even number"); }
        else { Console.WriteLine($"{number} is odd number"); }
    }
}